lib/tasks/gather.js

```diff
@@ -35,6 +35,7 @@ class TaskGather extends SttTask {
 
     /* timeout of zero means no timeout */
     this.timeout = this.timeout === 0 ? 0 : (this.timeout || 15) * 1000;
+    this.interDigitTimeout = (this.interDigitTimeout > 0 ? this.interDigitTimeout : 0) * 1000;
```

Comment:
was this changed needed to fix this bug?  Or is it an unrelated fix?

Reply:
this is to make sure there is value for interDigitTimeout that later we can do compare this. interDigitTimeout <= 0.

But I think this is not needed.

I added new changes and retested on jambonz.me

Reply:
Yes I have a feeling we are over-complicating things here.  I will have more time to review later today

Reply:
where are we with this PR @xquanluu ?